CompilerSource/ipc/Room.fbs

```diff
@@ -0,0 +1,113 @@
+
+table RoomInstance {
+  id:int;
+
+  x:int;
+  y:int;
+  object_id:int;
+
+  flip_x:bool;
+  flip_y:bool;
+  scale_x:double;
+  scale_y:double;
+  rotation:double;
+  color:int;
+
+  creation_code:string;

```

Comment:
So there was a pre_creation_code before in our old backend that does not exist in GMS. I believe that was because it was easier for you and egofree to add color and rotation that way instead of updating the plugin. Since we have this now, I've just added the fields they would be since this mechanism is intended to be easier to update in the future. Or do you still find pre_creation_code kind of useful enough to include it too?